modules/data-loading/pages/spark-connection-via-jdbc-driver.adoc

```diff
@@ -20,10 +21,288 @@ If you use spark job to connect TigerGraph via JDBC, we recommend your concurren
 This limits the concurrent JDBC connections to 40.
 
 ----
-—num-executors 2  /* 2 executors per job */
-—executor-cores 2 /* 1 executor take 2 cores */
+—-num-executors 2  /* 2 executors per job */
+—-executor-cores 2 /* 1 executor take 2 cores */
 ----
+====
+
+== Load From a Data Lake Via Spark
+
+=== Prerequisites: General Loading from Spark
```

Comment:
I think this section is not "Prerequisites" but the actual instructions for how to do basic Spark loading.
Also, the 5 bullets points should be "write a loading job" (1) which includes these 4 details.  So the hierarchy/structure of presenting that info might be done a little differently.

Reply:
Thank you these have been addressed with latest commit.